### ISSUE TITLE
Ctrl+Z

### ISSUE THREAD

**Issue by fenykep** [fenykep/pyproject#5]
We could create a temp var that stores the variables that the user deletes or modifies so they can revert a step.

**Comment by MinasMayth** [comment]
You mean like a rollback function? I think there's something in SQLite for that actually

**Comment by fenykep** [comment]
Yeah, great. But it's not a priority, I'm trying to write the gui app object oriented
